src/test/java/org/apache/commons/io/output/ByteArrayOutputStreamTest.java

```diff
@@ -350,6 +360,22 @@ public void testWriteZero(final String baosName, final BAOSFactory<?> baosFactor
         }
     }
 
+    private static final int FILE_SIZE = (1024 * 4) + 1;
+    private final byte[] inData = TestUtils.generateTestData(FILE_SIZE);
```

Comment:
These data do not need to be tracked for all tests. Move them inside the one method that use them.